packages/icons/saltIcons.css

```diff
@@ -0,0 +1,1473 @@
+/** WARNING: This file includes all icons in a css mask format and was generated by a script. Do not modify it manually */
+[class*=" saltIcon-"],
+[class^="saltIcon-"] {
```

Comment:
this causes problems if you set the color to primary or secondary (to test, add color='primary' to all icons QA story)
I assume the use case for using the icon component directly vs the class name would mean this rarely occurs, someone wouldn't import the css if using the component directly, but something to consider

Reply:
good spot!! will change the classname